### DISCUSSION TITLE
Problem changing the file to another directory

### DISCUSSION THREAD

**Discussion by robertolccj** [xournalpp/xournalpp#3567] [Q&A]
I have a problem when I change a file to another directory. A mensage show complete old path and does not find the background file (pdf). How can I configure the software to search the same filename on own directory where original file is? In that case, for default, the backgroud file and the xournal file would be in same directory and with the same name.

**Comment by cflange** [comment]
I have a similar problem. I synchronize project directories between desktop and laptop, but the laptop has a  different full path, because I place the projects in an encrypted directory. Is there a way to make the background pointer use a relative path, instead of the full path? This would solve the OP problem and mine at the same time.

**Comment by Vonpiper** [comment]
I too had this issue.  This feature was added here, when creating new files:
https://github.com/xournalpp/xournalpp/pull/2106

To modify my existing xopp files to use relative paths, I used the methods described here:
https://github.com/xournalpp/xournalpp/issues/3189
